components/user-service/src/main/java/edu/jhuapl/dorset/fileuserservice/FileUserService.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2016 The Johns Hopkins University Applied Physics Laboratory LLC
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.jhuapl.dorset.fileuserservice;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import edu.jhuapl.dorset.users.User;
+import edu.jhuapl.dorset.users.UserException;
+import edu.jhuapl.dorset.users.UserService;
+
+public class FileUserService implements UserService {
+    private static final Logger logger = LoggerFactory.getLogger(FileUserService.class);
+
+    private Map<String, User> users;
+    protected String userDirPath;
+    protected String fileBaseName;
+
+    /**
+     * 
+     * FileUserService
+     * 
+     * @param userDirectoryPath Path to directory that contains User files
+     * @param fileBaseName Base string of User file names
```

Comment:
Just from the documentation, I cannot figure out what this means. Maybe add some class develop documentation that explains how this works with files/directories.